components/tools/ToolDashboard.js

```diff
@@ -61,6 +61,22 @@ export default function ToolDashboard() {
     };
   });
 
+  // Navigates the user to the specific categories on providing the URL
+  useEffect(() => {
+    if (!loading) {
+        const urlPath = router.asPath;
+        const elementIndex = urlPath.lastIndexOf('#');
+        if (elementIndex !== -1) {
+          const element = urlPath.substring(elementIndex + 1).replace(/%20/g, ' ');
```

Comment:
I don't think we should use hardcode functions.
Is there any other way of doing this??

Reply:
I didn't get what you mean by hardcoded functions. Could you please elaborate on it?

Reply:
We are looking at the specific component of the URL i.e. the category. Hence for URLs like ``domain/tools#Frameworks`` I try to extract the word ``Frameworks`` using ``elementIndex``.

Reply:
try running this url on your local host `https://localhost:3000/tools#Documentation%20Generators`

Reply:
https://github.com/asyncapi/website/assets/100564488/f635e6c1-19a3-4dd8-9f59-bf60b0c2abf8

Reply:
@sambhavgupta0705 is the PR good to be merged?